core/imageroot/etc/systemd/system/phonehome.timer

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=PhoneHome Trigger
+
+[Timer]
+OnCalendar=daily
```

Comment:
IIRC Systemd allows to add a random offset. Could you use it to avoid all clients send their payload at the same time?

Reply:
Was not aware of the random offset, will do.